doc/source/development/contributing_environment.rst

```diff
@@ -249,6 +249,11 @@ To compile pandas with setuptools, run::
 
    python setup.py develop
 
+.. note::
+   If pandas is already installed, you have to uninstall it first::
+
+        pip uninstall pandas
+
```

Comment:
I think it is good to add the context for why you should uninstall it.

e.g.
"This is because ``python setup.py develop`` will not uninstall the loader script that meson-python uses to import the extension from the build folder, which may cause errors such as an ``FileNotFoundError`` to be raised."

Reply:
done